cmake/EDM4HEPConfig.cmake.in

```diff
@@ -6,6 +6,7 @@
 # - Create relocatable paths to headers.
 # NOTE: Do not strictly need paths as all usage requirements are encoded in
 # the imported targets created later.
+set_and_check(EDM4HEP_DATA_DIR "@PACKAGE_CMAKE_INSTALL_DATADIR@/edm4hep")
```

Comment:
Can you move this below this block for INCLUDE_DIR, because the comment does not apply to DATADIR